webpack.config.js

```diff
@@ -0,0 +1,38 @@
+const path = require('path');
+
+module.exports = {
+  entry: path.resolve(__dirname, './client/src/index.js'),
+  output: {
+    filename: 'main.js',
+    path: path.resolve(__dirname, './client/dist'),
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
+        exclude: /node_modules/,
+        use: {
+          loader: "babel-loader",
+          options: {
+            presets: ['@babel/preset-env', '@babel/preset-react']
+          }
+        }
+      },
+      {
+        test: /\.css$/i,
+        use: ["style-loader", "css-loader"],
+      },
+      {
+        test: /\.svg$/,
+        use: 'file-loader',
+      },
+      {
+        test: /\.(s(a|c)ss)$/,
+        use: ['style-loader','css-loader', 'sass-loader']
+      }
+    ],
+  },
+  resolve: {
+    extensions: ['*', '.js', '.jsx'],
+  }
+};
```

Comment:
nit: can you add a blank line in the end of files to avoid the "no new line at end of file" github alert?